lib/spack/spack/installer.py

```diff
@@ -391,11 +391,11 @@ def _process_binary_cache_tarball(
         bool: ``True`` if the package was extracted from binary cache,
             else ``False``
     """
+    timer and timer.start("fetch")
```

Comment:
Not sure how idiomatic this is line is. It is effectively equal to:
```python
if timer:
    timer.start("fetch")
```
Would it be easy to code a `NullTimer` with the same interface but that performs no-op? If it is we can just:
```python
timer.start("fetch")
```
and set a NULL_TIMER object as the default value.

Reply:
Let me do null timer